backend/src/schema/resolvers/reports.js

```diff
@@ -11,12 +11,31 @@ export default {
         description: description,
       }
 
+      const reportQueryRes = await session.run(
+        `
+        match (u:User {id:$submitterId}) -[:REPORTED]->(report)-[:REPORTED]-> (resource {id: $resourceId}) 
+        return  labels(resource)[0] as label
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:47:16Z" title="Friday, June 21st 2019, 2:47:16 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,12 +11,32 @@ export default {
         description: description,
       }
 
+      const reportQueryRes = await session.run(
+        `
+        match (u:User {id:$submitterId}) -[:REPORTED]->(report)-[:REPORTED]-> (resource {id: $resourceId}) 
+        return  report, labels(resource)[0] as type
```


Nodes can have multiple labels, So I would suggest this
```suggestion
        return  report, labels(resource) as labels
```

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:47:50Z" title="Friday, June 21st 2019, 2:47:50 pm +02:00">Jun 21, 2019</time>_

---


Simply omitting the labels for now is also totally acceptable. As far as I can see, you're not using the labels anywhere.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-06-21T14:09:11Z" title="Friday, June 21st 2019, 4:09:11 pm +02:00">Jun 21, 2019</time>_

---


yes you are right .. where I now exactly overlook it .. would check a pure and return the same do . with a few lines less :)